### ISSUE TITLE
Is it possible to show three columns?

### ISSUE THREAD

**Issue by todatamining** [pomber/github-history#54]
column1 --------------------------column2 ------------------------------------------column3
code in commit1             code in commit2                           change in commit3

currently, only code in commit2 is shown. It is hard to know what has been changed or changed to exactly.

**Comment by pomber** [comment]
Not sure about this one. One of the things people like about this tool is that there isn't too much noise.  
#63 will add a link to GitHub commit where you can see the diff, maybe that's enough, or maybe we can add a special diff view later.

**Comment by shayanb** [comment]
I agree. The minimal view is really clear and perfect for finding the diffs. The url to the actual commit can help to investigate the details and change log.

**Comment by pomber** [comment]
From #22 
> The current visualization is more for browsing different versions of a file rather than viewing the diff between two versions. I want to keep that, but I agree that people usually want to see a diff after browsing the history.
> I'm thinking on adding some way to activate a "diff mode" that animates between diffs, as you change the "left" and/or the "right" version of a file.

**Comment by katrinleinweber** [comment]
How about only 2 columns? [git-temporal](https://marketplace.visualstudio.com/items?itemName=bee.git-temporal-vscode&ssr=false) does this. Would that be a good compromise between the 2 use-cases?
